File: Python Exams/Basic Exam Collection/Basics Online Exam -  28 and 29 March 2020/05. Suitcases Load/05. Suitcases Load.py
contents = float(input())
goods = 0
count_luggage = 0

while True:
   luggage = input()
   
   if luggage == 'End' or contents < goods:
     if luggage == 'End':
       print('Congratulations! All suitcases are loaded!')
     else:
       print('No more space!')
       count_luggage -= 1  
     break
   else:
     if count_luggage % 3 == 0:
        goods += float(luggage) * 1.10
     else:
        goods += float(luggage)
  
   count_luggage += 1

print(f'Statistic: {count_luggage} suitcases loaded.')

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam -  28 and 29 March 2020/05. Suitcases Load/05. Suitcases Load.js
function suitcasesLoad(arg) {
  let capacity = arg.shift()
  let luggage = arg.shift()
  let count = 0

  while (luggage !== 'End') {
    count++
    count % 3 == 0 ? capacity -= luggage * 1.10 : capacity -= luggage
   
    if (capacity < 0) {
      count--
      return `No more space!\nStatistic: ${count} suitcases loaded.`
    }

    luggage = arg.shift()
    
  }

  return `Congratulations! All suitcases are loaded!\nStatistic: ${count} suitcases loaded.`
}

// console.log(suitcasesLoad([550, 100, 252, 72, 'End']))